spec/models/llm_api_key_spec.rb

```diff
@@ -1,18 +1,27 @@
 require 'rails_helper'
 
 RSpec.describe LlmApiKey, type: :model do
+  let(:plain_api_key) { "plain_text_key_example" }
+  let(:api_key_encrypter) { instance_double(ApiKeyEncrypter) }
+  let(:ciphertext_blob) { "encrypted_api_key" }
+  let(:base64_ciphertext) { Base64.encode64(ciphertext_blob) }
```

Comment:
```suggestion
  let(:base64_ciphertext) { "ダミー文字列" }
```

これはテスト用の Mock なので、わざわざ正しく計算する必要はありません。
検証可能な値が返せれば十分です。